java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -450,7 +464,406 @@ open class KotlinFileExtractor(
         extractDeclInitializers(c.declarations, false) { Pair(blockId, obinitId) }
     }
 
-    val jvmStaticFqName = FqName("kotlin.jvm.JvmStatic")
+    // Adapted from RepeatedAnnotationLowering.kt
```

Comment:
This file is already quite large, can we easily move these adapted functions into a separate file?